homeassistant/components/daikin/sensor.py

```diff
@@ -26,6 +26,8 @@
 async def async_setup_entry(hass, entry, async_add_entities):
     """Set up Daikin climate based on config_entry."""
     daikin_api = hass.data[DAIKIN_DOMAIN].get(entry.entry_id)
+    if not daikin_api.device.support_outside_temperature:
+        SENSOR_TYPES.pop(ATTR_OUTSIDE_TEMPERATURE, None)
```

Comment:
We shouldn't modify the constant, I think, right?

Reply:
It was the easiest fix to just create an indoor sensor here at L33.

Reply:
You can make a deep copy and then modify that.

Reply:
I could do this instead:

```python
sensors = { ATTR_INSIDE_TEMPERATURE: SENSOR_TYPES[ATTR_INSIDE_TEMPERATURE] }
if daikin_api.device.support_outside_temperature:
    sensors[ATTR_OUTSIDE_TEMPERATURE] = SENSOR_TYPES[ATTR_OUTSIDE_TEMPERATURE]
```

Reply:
Yes!